sdk/packages/eternum/src/constants/resources.ts

```diff
@@ -844,6 +844,8 @@ export enum QuestType {
   Pillage = 15,
   Hyperstructure = 16,
   Contribution = 17,
+  PauseProduction = 18,
+  CreateDefenseArmy = 19,
```

Comment:
Can remove all numbers except for the first one, the rest will be incremented by one